Nef_3/include/CGAL/Nef_3/SNC_io_parser.h

```diff
@@ -356,7 +356,7 @@ class sort_sface_cycle_entries : public SNC_decorator<T> {
   typedef typename T::Vector_3          Vector_3;
 
  public:
-  sort_sface_cycle_entries(T D) : Base(D) {}
+  sort_sface_cycle_entries(T& D) : Base(D) {}
```

Comment:
```suggestion
  sort_sface_cycle_entries(const T& D) : Base(D) {}
```

Reply:
```
error: no matching constructor for initialization of 
'SNC_decorator<CGAL::SNC_structure<CGAL::Epeck, CGAL::SNC_indexed_items, bool> >'
  sort_sface_cycle_entries(const T& D) : Base(D) {}
                                         ^    ~
  ```

Reply:
One would need to add something like:
```diff
   SNC_decorator(SNC_structure& W)
     : Base(W), sncp_(&W) {}
+  SNC_decorator(const SNC_structure& W)
+    : Base(W), sncp_(const_cast<SNC_structure*>(&W)) {}
  ```
   But something tells me this is not the right solution 🤔